lochness/utils/path_checker.py

```diff
@@ -49,7 +49,7 @@ def update_eeg_check(df: pd.DataFrame) -> None:
     eeg_index = df[df['modality'] == 'EEG'].index
     eeg_df = df.loc[eeg_index]
     eeg_df['file_check'] = eeg_df['file_name'].str.match(
-            '[A-Z]{2}\d{5}_eeg_\d{4}\d{2}\d{2}\.*')
+            '[A-Z]{2}\d{5}_eeg_\d{4}\d{2}\d{2}.*.zip$')
```

Comment:
Why not just `*.zip`?

Reply:
What do you mean just `*.zip`? Without the date digit checking part?

Reply:
Previously it was `\.*`. Why is it not `\*.zip` now?

Reply:
There is an additional `.` to the left of `*` in your proposal. I am saying, that `.` is probably redundant.

Reply:
In the SOP, if the eeg scan is repeated, suffix of `_1` will be attached after the file name. `.*` is to grab this.